qualang_tools/video_mode/README.md

```diff
@@ -0,0 +1,184 @@
+original parameter list by operating an appropriate casting for all parameters that were provided by the user.
+
+To be able to access those newly created QUA variables later in the design of the program, we can use 
+```video_mode["param_name"]``` for each parameter loaded initially in the dictionary. We can then use those variables 
+as we please within the program, as shown above with the custom parameters used for the QUA statements.
+
+### Updating variables dynamically
+
+
+The ```VideoMode```, once launched in Python, will query continuously the user to input (through the keyboard) 
+the name of parameter, and the associated new value it should take.
+The format should be written as follows:
+```
+param_name=new_value
+```
+The user can also update a parameter that is a QUA array by providing a list of values separated by a space:
+```
+param_name= new_value_1 new_value_2 new_value_3 ...
+```
+
+Once the user has typed an appropriate update, the ```VideoMode``` will use the IO variables of the ``QuantumMachine`` 
+to pass the following information:
+- which parameter of the initial dictionary should be updated next (IO1)
+- what new value should be set for the parameter (IO2)
+
+For updating a QUA array, a loop over the elements of the array is performed to update dynamically each element 
+with a successive back and forth interaction between the program and the Python (through interleaved ```pause()``` within QUA and ```job.resume()``` in Python).
+
+As everything happens behind the scenes, the user does not need to worry about the details of the implementation.
+The only thing that the user should implement in the QUA program on top of the declaration of variables shown above is the following macro:
+```
+with program() as video_mode_prog:
+    ...
+    with infinite_loop():
+        video_mode.load_parameters()
+        ...
+```
+
+### Accessing QUA variables from ```VideoMode``` within the program
+The QUA variables declared in the program through the ```VideoMode.declare_variables()``` can be accessed from the
+through the ```video_mode["param_name"]``` syntax. This enables you to choose where your QUA variable should land in 
+your program. For instance, if you want to use the same QUA variable for two different pulse parameters, you can do the following:
+```
+with program() as video_mode_prog:
+    video_mode.declare_variables()
+    
+    with infinite_loop():
+        video_mode.load_parameters()
+        
+        play("my_pulse"*amp(video_mode["amp_param"]), "qe1")
+        play("my_other_pulse"*amp(video_mode["amp_param"]), "qe2")
+```
+
+Note that you can also access to the full list of QUA variables declared in the program through the ```video_mode.variables``` attribute.
+
+Alternatively, you can also access the QUA variables later in the code as follows:
+```
+with program() as video_mode_prog:
+    amp_param1, amp_param2, ... = video_mode.declare_variables()
```

Comment:
Is the ordered guaranteed according to the dict order?
I would guess so, but it needs to be stated